cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
I think it's safe to assume that we'll run the server on `localhost:3000`.
This way we don't need to put `http://localhost:3000` in our `cy.visit` commands. 😄

Reply:
Yeah, good idea.